### DISCUSSION TITLE
Download most Recent X?

### DISCUSSION THREAD

**Discussion by HeWhoRoams** [ErrorFlynn/ytdlp-interface#84] [Ideas]
First off, this is totally meeting my needs to store some videos both for myself and my kids. Thank you!

A lot of the creators I'm looking at don't 'season' their content via playlist, so I wonder how hard it would be to put in a playlist URL and specifiy download the most recent X videos?  For some of these it doesn't make sense to download the entire library, and really I just want to stay current.

**Comment by ErrorFlynn** [comment]
You can do that with the existing feature that allows you to select which videos to download from the playlist. Right-click on the queue item, and choose `Select videos` from the menu. Then you can select a range of videos either at the beginning or at the end of the list (depending on how the uploader sorted the playlist).

**Comment by HeWhoRoams** [comment]
Ah thank you, that works. Ideally I don't want to rely on the uploader having a sense for file organization or seasons or dating
